src/app/header/header.component.ts

```diff
@@ -17,26 +25,42 @@ import { StoreService } from '../store.service';
     <a *ngIf="(store.user$ | async) != null" routerLink="/" (click)="logout()">Изход</a>
     -->
 
+    <!--
     <div class="toggle border-primary br-md pointer" (click)="showSidebarChange.emit(!showSidebar)">
       <img class="toggle__image" [src]="showSidebar ? 'assets/chevron-up.svg' : 'assets/chevron-down.svg'" />
     </div>
+    -->
```

Comment:
Have you considered how the mobile design would work if there's no toggle button?

Reply:
Yep, I'll try to work it out like this:

![image](https://user-images.githubusercontent.com/5939646/34153636-6e0d0fc8-e4bb-11e7-80d6-f25486a2e9f9.png)

Reply:
This looks great! 😄